engine/apps/api/serializers/user.py

```diff
@@ -71,7 +67,7 @@ class Meta:
             "email",
             "username",
             "name",
-            "role",  # LEGACY.. this should get removed eventually
+            "role",
```

Comment:
I think this was improperly marked as "LEGACY" during the RBAC work. `role` is still needed for the foreseeable future.